sites/public/src/ListingView.tsx

```diff
@@ -74,13 +72,145 @@ export const ListingView = (props: ListingProps) => {
     unitType: t("t.unitType"),
     rent: t("t.rent"),
     availability: t("t.availability"),
+    ami: t("listings.unit.ami"),
   }
 
   let groupedUnits: Record<string, React.ReactNode>[] = null
-  if (listing.unitsSummary !== undefined && listing.unitsSummary.length > 0) {
-    groupedUnits = getSummariesTableFromUnitsSummary(listing.unitsSummary)
-  } else if (listing.unitsSummarized !== undefined) {
-    groupedUnits = getSummariesTableFromUnitSummary(listing.unitsSummarized.byUnitTypeAndRent)
+  let hmiHeaders: TableHeaders
+  let hmiData: Record<string, React.ReactNode>[] = null
+  if (listing.unitGroups !== undefined && listing.unitGroups.length > 0) {
+    // unit group summary
+    groupedUnits = listing.unitSummaries.unitGroupSummary.map((group) => {
+      let rentRange = null
+      let rentAsPercentIncomeRange = null
+      if (group.rentRange && group.rentRange.min === group.rentRange.max) {
+        rentRange = group.rentRange.min
+      } else if (group.rentRange) {
+        rentRange = `${group.rentRange.min} - ${group.rentRange.max}`
+      }
+
+      if (rentRange) {
+        rentRange = (
+          <span>
```

Comment:
Can we just do this on line 89

Reply:
We'd also have to do it on line 87, if we did that.